[2021-08-04 13:30:34,276] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_C 2021-08-04T13:30:29.552189+00:00 [queued]>
[2021-08-04 13:30:34,337] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: my_dag.training_model_C 2021-08-04T13:30:29.552189+00:00 [queued]>
[2021-08-04 13:30:34,337] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2021-08-04 13:30:34,338] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2021-08-04 13:30:34,339] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2021-08-04 13:30:34,368] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): training_model_C> on 2021-08-04T13:30:29.552189+00:00
[2021-08-04 13:30:34,400] {standard_task_runner.py:52} INFO - Started process 10578 to run task
[2021-08-04 13:30:34,434] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'my_dag', 'training_model_C', '2021-08-04T13:30:29.552189+00:00', '--job-id', '1725', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmp66d4x_4e', '--error-file', '/tmp/tmp81dxyafb']
[2021-08-04 13:30:34,435] {standard_task_runner.py:77} INFO - Job 1725: Subtask training_model_C
[2021-08-04 13:30:34,688] {logging_mixin.py:104} INFO - Running <TaskInstance: my_dag.training_model_C 2021-08-04T13:30:29.552189+00:00 [running]> on host 04479ea62028
[2021-08-04 13:30:34,977] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=my_dag
AIRFLOW_CTX_TASK_ID=training_model_C
AIRFLOW_CTX_EXECUTION_DATE=2021-08-04T13:30:29.552189+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-08-04T13:30:29.552189+00:00
[2021-08-04 13:30:34,978] {python.py:151} INFO - Done. Returned value was: 4
[2021-08-04 13:30:35,090] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=my_dag, task_id=training_model_C, execution_date=20210804T133029, start_date=20210804T133034, end_date=20210804T133035
[2021-08-04 13:30:35,293] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2021-08-04 13:30:35,326] {local_task_job.py:149} INFO - Task exited with return code 0
